ios/RNBranch.m

```diff
@@ -75,6 +92,10 @@ + (BOOL)requiresMainQueueSetup {
 + (void)setupBranchInstance:(Branch *)instance
 {
     RCTLogInfo(@"Initializing Branch SDK v. %@", BNC_SDK_VERSION);
+    if (!branchSdkVersionValid()) {
+        RCTLogError(@"Please use at least v. %@ of Branch. In your Podfile: pod 'Branch', '~> %@'. Then pod install.", MINIMUM_REQUIRED_BRANCH_SDK, MINIMUM_REQUIRED_BRANCH_SDK);
```

Comment:
RCTLogError generates the overlay at least in dev mode and renders the app useless. This can be reduced to a warning, but perhaps because this happens before JS loads, the warning did not show up in the app. Warning overlays can be dismissed and temporarily ignored.